packages/commercetools/composables/src/useCart/currentCart.ts

```diff
@@ -1,14 +1,14 @@
-import { createCart, getMe } from '@vue-storefront/commercetools-api';
+import { Context } from '@vue-storefront/core';
 
-const loadCurrentCart = async (customQueryFn = (user = null, cart = null) => ({ cart, user })) => {
+const loadCurrentCart = async (context: Context, customQueryFn = (user = null, cart = null) => ({ cart, user })) => {
```

Comment:
we shouldn't have access to full `context` here, just `api`. Otherwise the same code will have different `context` on different enviroments and this could easily lead to issues

Not to mentions integrators who will most likely try to make use of nuxt ctx here